### ISSUE TITLE
Scrolling

### ISSUE THREAD

**Issue by Jace25** [alanwoolley/CorsixTH-Android#109]
Since the last update scrolling seems very broken. With the two finger scrolling before it was possible to scroll while moving a desk or something else around. If you try this know, it will most time place the object at the location where you have touched for scrolling.

Also scrolling seems not be possible, while in build mode. This makes it difficult to build big rooms like research.

**Comment by alanwoolley** [comment]
Scrolling in the latest version is now with 1 finger rather than 2. With this in mind, is it still problematic for you?

**Comment by Jace25** [comment]
Yeah. Just try to scroll while moving a doctor around or while placing objects. 

Scrolling seems to be completely disabled while in build mode. One Finger does only move around objects or placing them, to Finger starts zooming.

While moving a doctor as example from the school room to the research lab you have to choose places for the scrolling touch where it is disallowed to place personal. Orherwise it would be placed there

Bzw: i am very happy your great work. I am playing with corsixth since years!

**Comment by alanwoolley** [comment]
OK - I think I understand. I'll try to see if there's any easy improvements that can be made in the future.  The controls have always been a tricky issue as CorsixTH doesn't have any native understanding of touch which is obviously a bit different to point+click. I had to rework the two-finger scrolling into one-finger in order to support zooming.

Thanks - nice to hear :)

**Comment by CrippledGamerMax** [comment]
I got the app on a nvidia shield k1 which works perfectly apart from scrolling, when in build mood I try to place a object but everything scroll so it’s very hard, even tried the beta tester one but no look, you use to scroll  with 2 fingers but now it’s one. Can this be fixed?
